src/tests/suite/daos_test.h

```diff
@@ -439,7 +439,17 @@ test_rmdir(const char *path, bool force)
 		if (!strcmp(ent->d_name, ".") || !strcmp(ent->d_name, ".."))
 			continue;   /* skips the dots */
 
-		sprintf(fullpath, "%s/%s", path, ent->d_name);
+		rc = snprintf(fullpath, PATH_MAX, "%s/%s", path, ent->d_name);
+		if (rc < 0 || rc >= PATH_MAX) {
+			if (rc < 0)
+				rc = -DER_INVAL;	/* invalid */
+			if (rc >= PATH_MAX)
+				rc = -DER_NOMEM;	/* overflow */
```

Comment:
Not really ENOMEM.  I think it's more INVAL.  Since it's an unexpected path length.